scripts/performance/common.py

```diff
@@ -150,9 +150,10 @@ def verbose(self) -> bool:
         '''Enables/Disables verbosity.'''
         return self.__verbose
 
-    def run(self, working_directory: str = None) -> None:
+    def run(self, working_directory: str = None) -> str:
         '''Executes specified shell command.'''
         should_pipe = self.verbose
+        output = ''
```

Comment:
nit: can make `output` a list (`output = []`), and join at the end of the function.

Reply:
This is a work in progress and not ready for review. I am just testing things at the moment.